hazelcast/src/test/java/com/hazelcast/internal/partition/MigrationInvocationsSafetyTest.java

```diff
@@ -193,4 +198,106 @@ public void run() {
         });
     }
 
+    @Test
+    public void migrationCommit_shouldBeRetried_whenTargetNotResponds() throws Exception {
+        Config config = getConfig(true, true)
+                .setProperty(GroupProperty.MAX_NO_HEARTBEAT_SECONDS.getName(), "5")
+                .setProperty(GroupProperty.HEARTBEAT_INTERVAL_SECONDS.getName(), "1")
+                .setProperty(GroupProperty.OPERATION_CALL_TIMEOUT_MILLIS.getName(), "3000");
+
+        final HazelcastInstance master = factory.newHazelcastInstance(config);
+        final HazelcastInstance slave1 = factory.newHazelcastInstance(config);
+        final HazelcastInstance slave2 = factory.newHazelcastInstance(config);
+
+        assertClusterSizeEventually(3, slave1, slave2);
```

Comment:
Same questions as in all tests so far :) Would this work?
```java
        assertClusterSizeEventually(3, factory.getAllHazelcastInstances());
        warmUpPartitions(factory.getAllHazelcastInstances());
```

Reply:
Same answer.